features/background-position.yml.dist

```diff
@@ -69,14 +69,14 @@ compat_features:
   - css.properties.background-position-y
 
   # baseline: high
-  # baseline_low_date: 2020-07-28
-  # baseline_high_date: 2023-01-28
+  # baseline_low_date: ≤2020-07-28
+  # baseline_high_date: ≤2023-01-28
   # support:
-  #   chrome: ≤83
-  #   chrome_android: "83"
-  #   edge: "83"
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "79"
   #   firefox: ≤72
-  #   firefox_android: "79"
+  #   firefox_android: ≤79
```

Comment:
This change doesn't make sense to me- shouldn't it be `≤72`?

Reply:
It makes sense because Fx Android 69-78 have been skipped. https://github.com/mdn/browser-compat-data/blob/8d1d3706247e66603e19205fcdd7852f956b35e3/browsers/firefox_android.json#L445